Include/cpython/unicodeobject.h

```diff
@@ -195,7 +195,11 @@ typedef struct {
 
 /* Use only if you know it's a string */
 static inline unsigned int PyUnicode_CHECK_INTERNED(PyObject *op) {
-    return _PyASCIIObject_CAST(op)->state.interned;
+#ifdef Py_GIL_DISABLED
+    return _Py_atomic_load_uint16_relaxed(&(_PyASCIIObject_CAST(op)->interned));
+#else
+    return _PyASCIIObject_CAST(op)->interned;
+#endif
```

Comment:
```suggestion
    return FT_ATOMIC_LOAD_UINT16_RELAXED(_PyASCIIObject_CAST(op)->interned);
```

Reply:
@kumaraditya303 IIRC, we can not use 'FT_ATOMIC_LOAD_UINT16_RELAXED' at Include/cpython/xxxx.h. Do we have to way for declare?

Reply:
Same issue at here: https://github.com/python/cpython/blob/180d417e9f9456defd4c5b53cae678c318287921/Include/cpython/listobject.h#L32-L35

Reply:
Ah, got it, somehow I thought it was internal header